guides/common/modules/con_unsupported-use-cases-for-deploying-foreman-on-aws.adoc

```diff
@@ -0,0 +1,13 @@
+= Use Cases that Do Not Work
```

Comment:
I think Ewoud recommended not including this section because some of the 'unusable' use cases will work with certain workarounds. Sections like this also create maintenance issues because they are never updated. In theory, we could put all the limitations for cloud providers in a single section.

This is just a comment, not a recommendation.

Reply:
This is out of scope for me follow-up PR. -> no change yet.